extensions/standard-processors/tests/unit/YamlConfigurationTests.cpp

```diff
@@ -456,7 +452,7 @@ Remote Process Groups:
 NiFi Properties Overrides: {}
       )";
   std::istringstream configYamlStream(TEST_CONFIG_YAML);
```

Comment:
this `configYamlStream` (and the one at line 497) are no longer needed, either

Reply:
removed